sprites/frog/frogCharacter.js

```diff
@@ -2,6 +2,7 @@
 const frogCharacter = {
   name:"frogCharacter",
   size: {width: 228, height: 288},
+  animationTypes: ['IDLE', 'WALK', 'EAT', 'CELEBRATE', 'DISGUST', 'ALL'],
```

Comment:
Instead of walk, I think hob is the right verb. In the same way I would say fly for a bird not walk.